### DISCUSSION TITLE
Porting Freecad 0.20 to Microsoft store?

### DISCUSSION THREAD

**Discussion by MisterMakerNL** [mayrinck/FOSSonMicrosoftStore#36] [Requests] [answered]
Hey, 
Hope you are doing well. 
I was testing Freecad 0.20 and had some installation issues, probably caused by MS so we start using their store.
So since not everyone is ready to move to Linux it is probably a good idea to upload Freecad 0.20 to the MS store.
This is the wiki: https://wiki.freecadweb.org/About_FreeCAD
The release notes are being worked on as we speak here:
https://wiki.freecadweb.org/Release_notes_0.20
Download at least the beta of 0.2 is now available here; 
https://github.com/FreeCAD/FreeCAD-Bundle/releases
There are a lot of paid copies on Windows Store and would be nice if we could stop them. I doubt the money of those shops goes to the devs.
I am not a dev btw but I will notify them here:
https://forum.freecadweb.org/viewtopic.php?f=9&t=68705

**Comment by mayrinck** [comment, marked as answer]
Hello @HaasNL2! I think it's better to wait for the program to become more stable, then we can work on that. It's a nice suggestion btw.

**Reply by MisterMakerNL** [reply]
Could you be a bit more specific?
If it's not stable enough for the windows store I wonder how stable those 2 older Freecad versions are that are being sold for 5 euro per license. 😉

**Reply by mayrinck** [reply]
1) If other people are building/porting the software to MS Store on their own and selling it _(which is horrible)_, I cannot stop them. That's not the objective here, so comparisons are not useful. (And about quality... You can probably guess that might be bad)

2) There is no reason to post software that is still a pre-release version. Softwares like this usually got bugs that could break the building of the software or don't allowthe upload to Microsoft Store. (The very own devs say: **This is a test build for FreeCAD 0.20. DON'T USE YET for productive work!**). Better wait until a stable release comes and then try with that one.

3) You'd better check out [this article](https://github.com/mayrinck/FOSSonMicrosoftStore/discussions/3) where is specified the needed stuff to start publishing a software, there's some things missing on your topic.

**Reply by MisterMakerNL** [reply]
I never intended to publish the beta I just asked if you could publish the full release when it is ready. 0.2 is kind of a big deal for us. 
I thought stopping people getting scammed out of their money is kind of a noble cause. 
I really like how you offered your help on github, I'm just not feeling those vibes here any more.
Tnx for making an effort too reply anyway!

**Comment by JuliaAgostino** [comment]
I'm pretty sure this project is no longer active. Don't get your hopes up.

**Reply by mayrinck** [reply]
Yeah, Time is short for me lately, I'm not putting much energy around here, still, is not dead... I've been searching a few things and will post about them soon.

**Reply by JuliaAgostino** [reply]
> Yeah, Time is short for me lately, I'm not putting much energy around here, still, is not dead... I've been searching a few things and will post about them soon.

Okay, that's good to know.
